cluster-controller/src/main/java/io/strimzi/controller/cluster/operations/openshift/DeleteImageStreamOperation.java

```diff
@@ -0,0 +1,65 @@
+package io.strimzi.controller.cluster.operations.openshift;
+
+import io.fabric8.openshift.api.model.ImageStream;
+import io.strimzi.controller.cluster.OpenShiftUtils;
+import io.vertx.core.AsyncResult;
+import io.vertx.core.Future;
+import io.vertx.core.Handler;
+import io.vertx.core.Vertx;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Deletes ImageStream resource
+ */
+public class DeleteImageStreamOperation extends OpenShiftOperation {
+    private static final Logger log = LoggerFactory.getLogger(DeleteImageStreamOperation.class.getName());
+    private final String namespace;
+    private final String name;
+
+    /**
+     * @param namespace
+     * @param name
+     */
+    public DeleteImageStreamOperation(String namespace, String name) {
```

Comment:
No name method (which is clear is a constructor) but not parameters description.